graviti/dataframe/frame.py

```diff
@@ -47,33 +59,52 @@ class DataFrame:
     """
 
     _T = TypeVar("_T", bound="DataFrame")
+    _data: List[Sequence[Any]]
 
     def __init__(
         self,
-        data: Union[Iterable[Any], Dict[Any, Any], "DataFrame", None] = None,
+        data: Union[Sequence[Sequence[Any]], Dict[str, Any], "DataFrame", None] = None,
         schema: Any = None,
         columns: Optional[Iterable[str]] = None,
-        client: Any = None,
     ) -> None:
-        pass
+        if data is None:
+            data = {}
+        if schema is not None:
+            # TODO: missing schema processing
+            pass
+        if columns is not None:
+            # TODO: missing columns processing
+            pass
+
+        if isinstance(data, dict):
+            self.columns = RowSeries(data)
+            self._data = self.columns._data
+            self.index = ColumnSeries(list(range(len(self._data[0]))))
+        else:
+            raise ValueError("DataFrame only support generate from dictionary now")
```

Comment:
`TypeError`?

Reply:
Pandas will raise `ValueError`  when use the wrong data to initialize.